### DISCUSSION TITLE
Suggest For Quality Armor

### DISCUSSION THREAD

**Discussion by Nao2706** [Lorenzo0111/QualityArmory#605] [Ideas]
- A list of worlds in which pvp with weapons is allowed or not (I tried using world guard but it did not block damage between players)

- when you use shift to aim and try to give a headshot this is effective around 30 blocks but if it goes beyond that range even if you aim at a mob/player the projectile will deviate from the center I don't know if I could fix that

- if possible add weapon variants with silencers.

- (I don't know how useful this suggestion can be) that when firing a weapon without a silencer it makes mobs around the player go towards him.

- if possible add weapons/others such as (Javelin, Explosive Mines, Claymore, Flares) And I think that would be all in terms of things I could mention if they could be added.

I had already mentioned it in the review of the plugin but they told me to mention it on github too

**Comment by Lorenzo0111** [comment]
Moved to #610
